### PR TITLE
Redo DB-10968 SET_GLOBAL_DATABASE_PROPERTY should check existence of properties, restrict execution of GET_REGION_SERVER_CONFIG_INFO

### PR DESCRIPTION
**Pull request by martinrupp**

### CODE REVIEW

**Comment by martinrupp** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/2436/

**Review by jyuanca — APPROVED** [review]

**Review by msirek — APPROVED** [review]

File: db-engine/src/main/java/com/splicemachine/utils/DbEngineUtils.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.splicemachine.utils;
```
Comment (arnaud-splice) [inline db-engine/src/main/java/com/splicemachine/utils/DbEngineUtils.java:1]: License missing

**Review by arnaud-splice — CHANGES_REQUESTED** [review]

**Review by carolp-503 — APPROVED** [review]
+1, but see Arnaud's comment about missing license file.

File: db-engine/src/main/java/com/splicemachine/utils/DbEngineUtils.java (hunk 1 above)
Comment (martinrupp) [reply db-engine/src/main/java/com/splicemachine/utils/DbEngineUtils.java:16]: done

File: db-shared/src/main/java/com/splicemachine/db/iapi/reference/PropertyHelper.java [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+            try {
+                if(field.getType().equals(String.class))
+                    propertyList.add( ((String)field.get(null)) );
+            } catch (IllegalAccessException e) {
+                // continue
+            }
+        }
+        return propertyList;
+    }
+}
```
[the remark this answers is not in the document]
Comment (martinrupp) [reply db-shared/src/main/java/com/splicemachine/db/iapi/reference/PropertyHelper.java:153]: done

**Comment by martinrupp** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/2471/

**Review by arnaud-splice — APPROVED** [review]

**Pull request merged**